asv_bench/benchmarks/utils/common.py

```diff
@@ -594,3 +594,10 @@ def prepare_io_data_parquet(test_filename: str, data_type: str, shapes: list):
         df.to_parquet(test_filenames[shape_id], index=False)
 
     return test_filenames
+
+
+def setup(*args, **kwargs):
+    # This function just needs to be imported into each benchmark file to
```

Comment:
Let's supplement this comment that ASV itself executes this `setup` function automatically.